configure.ac

```diff
@@ -2677,12 +2677,9 @@ AC_CHECK_LIB(sendfile, sendfile)
 AC_CHECK_LIB(dl, dlopen)	# Dynamic linking for SunOS/Solaris and SYSV
 AC_CHECK_LIB(dld, shl_load)	# Dynamic linking for HP-UX
 
-# only check for sem_init if thread support is requested
-if test "$with_threads" = "yes" -o -z "$with_threads"; then
-    AC_SEARCH_LIBS(sem_init, pthread rt posix4) # 'Real Time' functions on Solaris
+AC_SEARCH_LIBS(sem_init, pthread rt posix4) # 'Real Time' functions on Solaris
```

Comment:
Perhaps keep the alignment of comments at the right? Otherwise, LGTM.

Reply:
Done, thanks!